bin/runtime/src/lib.rs

```diff
@@ -966,6 +976,51 @@ impl_runtime_apis! {
             Executive::execute_block_no_check(block)
         }
     }
+
+    #[cfg(feature = "runtime-benchmarks")]
+    impl frame_benchmarking::Benchmark<Block> for Runtime {
+        fn benchmark_metadata(extra: bool) -> (
+            Vec<frame_benchmarking::BenchmarkList>,
+            Vec<frame_support::traits::StorageInfo>,
+        ) {
+            use frame_benchmarking::{Benchmarking, BenchmarkList};
+            use frame_support::traits::StorageInfoTrait;
+
+            let mut list = Vec::<BenchmarkList>::new();
+            list_benchmarks!(list, extra);
```

Comment:
can you pass just a vec![] here, since the list is empty anway?

Reply:
no, unfortunately the macro is expecting this way :P